### PR TITLE
Add What's new page for 1.8.0-Beta

### PR DESCRIPTION
**Pull request by sarahhaggarty**
A pull request to add a What's new page for 1.8.0-Beta.

### CODE REVIEW

File: docs/v.list [hunk 1]
```diff
@@ -18,15 +18,15 @@
 
     <var name="coroutinesEapVersion" value="1.6.0-RC3" type="string"/>
 
-    <var name="minGradleVersion" value="6.7.1" type="string"/>
+    <var name="minGradleVersion" value="6.7.3" type="string"/>
```
Comment (koshachy) [inline docs/v.list:21]: If these Gradle and AGP versions are applicable only to the Beta, we shouldn't change them.
These versions specified only for stable release (no `Eap` in the name of variable).

File: docs/v.list [hunk 2]
```diff
@@ -4,11 +4,11 @@
 
 <vars>
 
-    <var name="kotlinVersion" value="1.7.20" type="string"/>
```
Comment (koshachy) [inline docs/v.list:7]: kotlinVersion and LatestUrl vars should be udpated only when the stable 1.8.0 is out

File: .idea/vcs.xml [hunk 3]
```diff
@@ -7,6 +7,7 @@
   </component>
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/kotlinx-lincheck" vcs="Git" />
```
Comment (koshachy) [inline .idea/vcs.xml:10]: It shouldn't be there, I suppose?

File: docs/topics/whatsnew18.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+_[Release date: %kotlinReleaseDate%](eap.md#build-details)_
+
+The Kotlin 1.8.0 release is out! Here are some highlights from this release:
+
+* The old backend for Kotlin/JVM has been removed
+* We now support Xcode 14
+* We ensured compatibility with Gradle 7.3
+* We introduced new experimental functions for JVM: recursively copy or delete directory content
+
+## Kotlin/JVM
```
Comment (koshachy) [inline docs/topics/whatsnew18.md:16]: Let's move info from the change notes here?

## Kotlin/JVM

* Support for Java 19 bytecode
* Removed old backend (the `-Xuse-old-backend` compiler option is no longer supported)

etc..

**Review by koshachy — CHANGES_REQUESTED** [review]

File: docs/topics/whatsnew18.md (hunk 4 above)
Comment (koshachy) [reply docs/topics/whatsnew18.md:16]: Or should we do it later?

Comment (sarahhaggarty) [reply docs/topics/whatsnew18.md:16]: @koshachy, change notes content added!

File: docs/topics/whatsnew-eap.md [hunk 5]
```diff
@@ -1,238 +1,73 @@
...
-
-* The `kotlin.targets.fromPreset()` convention is deprecated. Instead, you can still use `kotlin.targets { fromPreset() }` approach, but we recommend using more [specialized ways to create targets](multiplatform-set-up-targets.md).
-* Target accessors auto-generated by Gradle are no longer available inside the `kotlin.targets { }` block. Please use the `findByName("targetName")` method instead.
-Note that such accessors are still available in the `kotlin.targets` case, for example `kotlin.targets.linuxX64`.
-
-## How to update to the Kotlin 1.7.20-RC
-
-The IDE support for Kotlin 1.7.20-RC is available for IntelliJ IDEA 2022.2.1, Android Studio Dolphin (2021.3.1),
-and Android Studio Electric Eel (2022.1.1).
+The Kotlin 1.8.0-beta release is out! Here are some highlights from this release:
```
Comment (koshachy) [inline docs/topics/whatsnew-eap.md:10]:
```suggestion
The Kotlin 1.8.0-Beta release is out! Here are some highlights from this release:
```

**Review by koshachy — APPROVED** [review]
Looks nice!
Please add new lines after the headings for each section.

**Pull request merged**